lib/custom_html_renderer.rb

```diff
@@ -0,0 +1,7 @@
+require 'middleman-core/renderers/redcarpet'
+
+class CustomHTMLRenderer < Middleman::Renderers::MiddlemanRedcarpetHTML
```

Comment:
Could it be e.g. TechDocsHTMLRenderer or something? I can see it expanding to cover all manner of sins in the future, but `CustomHTMLRenderer` does seem a bit too generic.